### DISCUSSION TITLE
Coupon expiration reminder

### DISCUSSION THREAD

**Discussion by dmusican** [community/community#27373] [GitHub Education]
Hi there,

I just received the below email, telling me that I was going to start being billed for GitHub Pro if I didn't apply for a new coupon:

> Thanks for using GitHub! A while ago you used a GitHub coupon that was good for $4.00 off 2 years of GitHub Pro . This coupon will expire in 14 days (August 10, 2022) and your account will be charged for the existing subscription on the next billing date. If you are still a student or teacher, you may apply for a new academic discount coupon at https://education.github.com/benefits. Since you already have a credit card on file you don’t need to take any additional actions. We just wanted to let you know that you’ll be charged full price($4/month) after your coupon expires.

I visited https://education.github.com/benefits, but there's nothing there that I can find about getting a new coupon for GitHub Pro. How can I go about doing so? If I can't find it, I assume that I need to make sure that I downgrade from GitHub Pro before the system starts charging me. Is that correct?

**Comment by MrSarno** [comment]
Hi @dmusican,

Could you clarify whether you are presently a student or teacher?

The Benefits page, linked to above, has links to forms usable by each to obtain complimentary GitHub Pro access.

**Comment by dmusican** [comment]
I'm a teacher. Here's what I see... the page is fairly empty.

![image](https://user-images.githubusercontent.com/695685/181493677-4ea95b82-9edf-45f8-8722-6434041d9ec0.png)

**Reply by MrSarno** [reply]
Could you try using a different web browser, @dmusican?

This is what I see;
![3011A8EF-2CED-4157-B954-3E5418EEC227](https://user-images.githubusercontent.com/32073555/181603762-9c53ece0-3c86-45c3-af2b-8afce98ca4e0.jpeg)

**Comment by dmusican** [comment]
@MrSarno: I see the same thing you do when I'm not logged in, but if I'm logged in, I see the page I showed earlier. This is true in multiple browsers.

So I next tried to click the "get teacher benefits" button when I wasn't logged in, which took me to this screen (which still remembered I was logged in, even though I had logged out, there's clearly some weird session persistence stuff going on). This screen shows that I'm up to date, but it doesn't say anything about GitHub Pro or a discount coupon, which is what the email indicated I should be looking for.

![image](https://user-images.githubusercontent.com/695685/181754978-af4e769f-43af-46ea-8e19-f40e4c54a5e2.png)

**Reply by kereid** [reply]
@dmusican has this been resolved for you yet if so what happened? I am having the same issue with our organisation discount and our "coupon" expired on the 12th of August?

**Reply by dmusican** [reply]
@kereid : I'm not sure. The above behavior I described remains the same. That said, when I go to my GitHub settings, then select "Billing and Plans," then "Update Payment Method," it does show "Coupon: $4.00 off for 2 years until July 26, 2024." So, hopefully that's good and the coupon applies. Is that coupon functioning, or do I need to redeem it somehow? It's just not clear. In the meantime, I removed my credit card from the payment methods, just in case.

**Reply by kereid** [reply]
@dmusican thanks for replying! Hopefully mine will do the same.

**Comment by adubey14** [comment]
This is a key problem with github systems. Even if you upload the faculty status documents, their auto system is unable to read anything and just keeps failing. What is the recourse for this.

**Reply by nicoguaro** [reply]
This is my problem as well. I get the following message:


> Your application cannot be reviewed until you fix the following:
>
> The text in the photo you used was unable to be read. If using your device camera, make sure the image is clear and legible. For some languages, you may need to capture the original document beside an English translation.
> 
> The image you selected does not appear to contain your school name. You may include multiple documents in your image, so long as they are legible.
> 
> You are more likely to be verified if you have completed your GitHub user profile.
> 
>  Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.

**Comment by mikima** [comment]
Hello, I'm encountering the same problem. I apply for the coupon, added all the requested information, but at the end it says that "Your application cannot be reviewed", and there is no way to send the application (see the screenshot below). Is there a solution to this issue?

<img width="755" alt="Screenshot 2022-09-06 at 09 47 57" src="https://user-images.githubusercontent.com/907536/188577268-48a2cdd5-9c82-4f07-ba74-c3d5a0719f96.png">

**Comment by Vigneshgvs** [comment]
https://education.github.com/discount_requests/application (scroll down) and fill out the form. Will work, if you upload ID with date or "Student/Teacher enrollment verification" form ur university (all univ do provide online).
